packages/compat/src/compat-app-builder.ts

```diff
@@ -391,20 +391,9 @@ export class CompatAppBuilder {
     for (let addon of sortBy(Array.from(engine.addons.keys()), this.scriptPriority.bind(this))) {
       let implicitScripts = addon.meta[type];
       if (implicitScripts) {
-        let styles = [];
         let options = { basedir: addon.root };
         for (let mod of implicitScripts) {
-          if (type === 'implicit-styles') {
-            // exclude engines because they will handle their own css importation
-            if (!addon.isLazyEngine()) {
-              styles.push(resolve.sync(mod, options));
-            }
-          } else {
```

Comment:
Note for myself: that may not be ready for clean-up after all, the function may be called with `implicit-styles` parameter somewhere else